docs/source/en-us/env.md

```diff
@@ -0,0 +1,39 @@
+title: Runtime Enviroment
+---
+
+# runtime enviroment
+
+There could be all kinds of difference during various stages of a web application development, but the application itself should be stateless, so EGG provide enviromental variables to cope with such difference.
```

Comment:
1. difference of...what?
2. enviromental variables -> environment variables
3. cope with -> deal with?